tools/toc.sh

```diff
@@ -9,7 +9,7 @@
 let ERROR=0
 
 # Find all markdown files
-for f in $(find . | grep .md); do
+for f in $(find doc/ boards/ kernel/ arch/ capsules/ chips/ userland/ -name "*.md"); do
```

Comment:
I don't really see piping to grep as bad?

The other thing I like is that with the `git` approach, it'll only touch files that are in version control. Even though this should be super safe, I'm always super wary about touching files automatically that aren't in version control :/